modules/elasticsearch-index/src/main/java/org/opencastproject/elasticsearch/index/endpoint/IndexEndpoint.java

```diff
@@ -133,7 +133,7 @@ public Response clearIndex() {
       description = "Repopulates the Index from an specific service",
       returnDescription = "OK if repopulation has started", pathParameters = {
       @RestParameter(name = "service", isRequired = true, description = "The service to recreate index from. "
-        + "The available services are: Themes, Series, Scheduler, AssetManager, Comments and Workflow. "
+        + "The available services are: Themes, Series, Scheduler, AssetManager, Comments, Search and Workflow. "
         + "The service order (see above) is very important! Make sure, you do not run index rebuild for more than one "
```

Comment:
The order is actually “Comments, Workflow and Search”. Also, Search is separate from the other index. AssetManager, Scheduler, Workflows, … are services which actually write to one index. That is why the order is important. Search is completely separate.

```suggestion
        + "The available services are: Themes, Series, Scheduler, AssetManager, Comments, Workflow and Search. "
        + "The service order (see above) is very important except for Search! Make sure, you do not run index rebuild for more than one "
```